addons/binding/org.openhab.binding.irtrans/src/main/java/org/openhab/binding/irtrans/handler/EthernetBridgeHandler.java

```diff
@@ -779,13 +778,13 @@ protected void onWritable(ByteBuffer buffer) {
 
     protected void onRead(ByteBuffer byteBuffer) {
         try {
-            logger.trace("Received bytebuffer : '{}'", DatatypeConverter.printHexBinary(byteBuffer.array()));
+            logger.trace("Received bytebuffer : '{}'", HexUtils.bytesToHex(byteBuffer.array()));
             int byteCount = getByteCount(byteBuffer);
 
             while (byteCount > 0) {
                 byte[] message = new byte[byteCount];
                 byteBuffer.get(message, 0, byteCount);
-                logger.trace("Received message : '{}'", DatatypeConverter.printHexBinary(message));
+                logger.trace("Received message : '{}'", HexUtils.bytesToHex(message));
```

Comment:
The `HexUtils.bytesToHex()` method is applied every time. Should we encapsulate it by an `if (logger.isTraceEnabled())`?